Source/Utils/Interpolate.cpp

```diff
@@ -69,7 +69,7 @@ namespace Interpolate
         IntVect fy_type = interpolated_F[1]->ixType().toIntVect();
         IntVect fz_type = interpolated_F[2]->ixType().toIntVect();
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
I agree with you

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```